protx-client/src/components/ProTx/components/charts/MainChart.jsx

```diff
@@ -10,10 +10,14 @@ import MaltreatmentDetails from './MaltreatmentDetails';
 import MainPlot from './MainPlot';
 import styles from './MainChart.module.scss';
 import { getSelectedGeographyName } from '../shared/dataUtils';
+import { Button } from 'reactstrap';
+import CommunityCharacteristicsButton from '../modals/CommunityCharacteristics';
```

Comment:
maybe call CommunityCharacteristics instead of CommunityCharacteristicsButton.  Or CommunityCharacteristicsButtonModal?